src/crosstheborder/ui/ClientMain.java

```diff
@@ -49,6 +50,10 @@ public void start(Stage primaryStage) {
 
     }
 
+    /**
+     * used to execute methods in this class from anywhere.
+     * @return this instance of the class
+     */
     public static ClientMain getInstance(){
         return thisInstance;
```

Comment:
This is not a good coding practice. Either pass on the ClientMain instance to the inner layout controllers or implement an observer pattern.